app/views/pages/index.html.haml

```diff
@@ -1 +1,6 @@
-Exercism!
+Hello, World!
+%hr/
+%h2 Temporary useful buttons
+%ul
+  %li
+    = link_to "Create Iteration", tmp_iterations_path, method: :post, remote: true
```

Comment:
This triggers the action above, which should get end-to-end through the test runner. it's pending one final callback in the orchestrator having the right URL atm. Then it'll be done 👍 

I'm thinking I talk you through adding that config tomorrow to familiarise you more with all this config stuff.

Reply:
👍